include/constants/items.h

```diff
@@ -1002,6 +1005,6 @@
 #define ITEM_B_USE_OTHER    2
 
 // Check if the item is one that can be used on a Pokemon.
-#define ITEM_HAS_EFFECT(item) ((item) >= ITEM_POTION && (item) <= LAST_BERRY_INDEX)
+#define ITEM_HAS_EFFECT(item) ((item) >= ITEM_POTION && (item) <= ITEMS_COUNT)
```

Comment:
Forgot to say, set this as `ITEM_PEAT_BLOCK` instead of `ITEMS_COUNT`, as it could cause issues down the line.